disnake/ui/select.py

```diff
@@ -304,6 +320,10 @@ def select(
     In order to get the selected items that the user has chosen within the callback
     use :attr:`Select.values`.
 
+    .. versionchanged:: 2.5
```

Comment:
I feel like this should be moved to the `options` parameter, otherwise you might miss it when reading through the documentation and only looking at that specific option
(same thing above)